spec/menu_spec.rb

```diff
@@ -0,0 +1,29 @@
+require './lib/menu'
+
+describe Menu do
+
+  describe '#initialize' do
+    it 'is initialized with an empty hash' do
+      menu = Menu.new
+      expect(menu.contents).to be_kind_of(Hash)
```

Comment:
Minor — this test could be changed/simplified to `expect(menu.contents).to eq({})` so it checks the type and *also* that the hash is empty